watchify/watchify.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for watchify v3.7.0
+// Project: https://github.com/substack/watchify
+// Definitions by: TeamworkGuy2 <https://github.com/TeamworkGuy2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../browserify/browserify.d.ts" />
+
+declare module 'watchify' {
+
+    var Watchify: Watchify.Constructor;
+
+    /** Watch mode for browserify builds.
+     * Update any source file and your browserify bundle will be recompiled on the spot
+     */
+    module Watchify {
```

Comment:
Oh really... I thought it was the other way round and namespace was the old way, will do in future. 
Is namespace preferred specifically for the declare namespace 'abc' type stuff that is used for creating definitions for imports or all .d.ts files in general?